tests/flows/erc20_to_native_token_bridge.go

```diff
@@ -63,21 +62,26 @@ func ERC20ToNativeTokenBridge(network interfaces.LocalNetwork) {
 	log.Info("Example ERC20 Contract Address: " + exampleERC20ContractAddress.Hex())
 
 	{
-		// Fund the deployer addresses on each chain
+		// Fund the deployer address with sufficient native tokens (100 eth = 1e20 wei) on the source chain to deploy the
+		// contract and send a number of transfer transactions.
+		sourceFundingAmount := utils.BigIntMul(big.NewInt(1e15), big.NewInt(1e5))
```

Comment:
Can we just use `big.NewInt(1e20)`?

Reply:
Nope, `1e19` is the largest that works I believe.  The scientific notation passes in an int64